infrastructure/modules/monitoring/fluent-bit/configs/input/input-kubernetes-daemonset.conf

```
@@ -24,4 +24,18 @@
     Mem_Buf_Limit      30MB
     Skip_Long_Lines    Off
     Refresh_Interval   10
-    READ_FROM_HEAD     ${READ_FROM_HEAD}
\ No newline at end of file
+    READ_FROM_HEAD     ${READ_FROM_HEAD}
+
+[INPUT]
+    Name               tail
+    Tag                s3-application.*
+    Path               /var/log/containers/control-plane*.log, /var/log/containers/compute-plane*.log, /var/log/containers/ingress*.log, /var/log/containers/mongodb*.log, /var/log/containers/keda*.log
+    Parser             cri
```

```suggestion
    Parser             docker
```
We will make the changes in the next PR.

---

done